README.md

```diff
@@ -1,7 +1,41 @@
 
 # income-tax-cis-frontend
 
-This is a placeholder README.md for a new repository
+This is where users can review and make changes to the CIS section of their income tax return.
+
+## Running the service locally
+
+you will need to have the following:
+- Installed [MondoDB](https://docs.mongodb.com/manual/installation/)
+- Installed/configured [service-manager](https://github.com/hmrc/service-manager)
+
+The service manager profile for this service is:
+
+    sm --start INCOME_TAX_EMPLOYMENT_FRONTEND
+Run the following command to start the remaining services locally:
+
+    sudo mongod (If not already running)
+    sm --start INCOME_TAX_SUBMISSION_ALL -r
+
+This service runs on port: `localhost:9338`
+
+### Feature Switches
+
+| Feature | Environments Enabled In |
+| --- | --- |
+| Encryption | None|
+| Welsh Toggle | QA, Staging |
+| NRS | Local, QA, Staging, ET, Production |
+| End of year employment | Production |
+
```

Comment:
Can you please add a section similar to 'CIS Sources (Contractor and Customer Data)' from the BE?